### DISCUSSION TITLE
Donation Option

### DISCUSSION THREAD

**Discussion by jethrop** [cline/cline#826] [Feature Requests]
Many, many people really appreciate this plugin.  I know I personally would like to donate to the cause.  Could you setup a donation system?  There are a few options, like Github Sponsors, Liberapay, Gitcoin, and OpenCollective.  Or patreon, of course.
https://liberapay.com/
https://github.com/sponsors
https://opencollective.com/
https://www.gitcoin.co/

**Comment by Shegox** [comment]
We, [at SAP](https://github.com/orgs/SAP/sponsoring), would like to support the open source development of Cline as well through direct funding.

For this we would ideally want to support the project (or individual maintainers) directly through [GitHub Sponsors](https://github.com/open-source/sponsors). 

If you're interested in setting it up, please let me know and I can start the payout. Alternatively you can as well reach us at ospo@sap.com (SAP Open Source Program Office).
